starrocks-stream-load-sdk/src/main/java/com/starrocks/data/load/stream/StreamLoadDataFormat.java

```diff
@@ -30,7 +30,7 @@ public CSVFormat(String rowDelimiter) {
 
         @Override
         public byte[] first() {
-            return NEW_LINE;
+            return null;
```

Comment:
// EMPTY_BYTE_ARR = new byte[0]
return EMPTY_BYTE_ARR; 

Is it better not to be null ?

Reply:
Fixed